src/core/ui/components/sign-in/screens/loading.tsx

```diff
@@ -0,0 +1,63 @@
+import { Header } from '../layout/header';
+import { Content, Section } from '../layout/content';
+import { Footer } from '../layout/footer';
+import { Text, Size } from '../../primitives/text';
+import { useLoginMethods, getIconsById } from '../../../context/login-methods';
+import { Flex } from '../../primitives/container';
+import { HighlightedIcon } from '../../primitives/icon';
+import { Loader } from '../../primitives/loader';
+import { BaseButton } from '../../primitives/button';
+import { classNames } from '../../../../../shared/utils/classnames';
+import text from '../../primitives/text.module.css';
+import margin from '../../primitives/margin.module.css';
+import styles from './loading.module.css';
+
+export const LoadingScreen = ({ navigateBack }) => {
+  const { selectedLoginMethod: selected } = useLoginMethods();
+
+  const selectedLoginMethod = selected;
+
+  return (
+    <>
+      <Header title="Confirm Connection" />
+      <Content>
+        <Section>
+          <Flex gap={8} alignItems="center" justifyContent="center">
+            <HighlightedIcon>
+              <img src={getIconsById('slashAuth')} alt="SlashAuth logo" />
+            </HighlightedIcon>
+            <Loader color="black" size={8} />
+            <HighlightedIcon>
+              <img
+                src={getIconsById(selectedLoginMethod?.id)}
+                alt={`${selectedLoginMethod?.name} logo`}
+              />
+            </HighlightedIcon>
+          </Flex>
+          <Text
+            className={classNames(text.centered, margin.top4)}
+            size={Size.Large}
+          >
+            Connecting{' '}
+            {selectedLoginMethod ? `to ${selectedLoginMethod?.name}` : ''}
+          </Text>
+          <Text
+            className={classNames(
+              text.centered,
+              margin.top2,
+              styles.detailedDescription
+            )}
+          >
+            Please sign the message in your wallet app or plugin to continue
```

Comment:
Given this is only for wallets do we want to have a different set of options or a different component for non-wallet logins?

Reply:
What would the texts look like for non-wallet logins?

Reply:
We discussed this morning -- we should update this but if it makes sense to do it in a future PR that's fine too!